frontend/src/static/js/v_summary.js

```diff
@@ -98,16 +98,27 @@ window.vSummary = {
     },
 
     filterGroupSelection() {
+      this.getFilteredRepos();
+
       // merge group is not allowed when group by none
+      // also reset merged groups
       if (this.filterGroupSelection === 'groupByNone') {
-        this.isMergeGroup = false;
+        this.mergedGroupsMap = {};
+      } else if (!this.allGroupsMerged) {
+        this.resetMergedGroupsMap();
+      } else {
+        this.resetMergedGroupsMap();
+        Object.keys(this.mergedGroupsMap).forEach((x) => {
+          this.mergedGroupsMap[x] = true;
```

Comment:
Why we have to use a map to store all the merged Groups? Why a list is not enough?

Reply:
I suppose using a map makes the check and modification easier.